frontend/index.html

```diff
@@ -8,35 +8,9 @@
   </head>
 
   <body>
-    <!-- The core Firebase JS SDK is always required and must be listed first -->
-    <script src="https://www.gstatic.com/firebasejs/7.17.2/firebase-app.js"></script>
-    
-    <!-- TODO: Add SDKs for Firebase products that you want to use
-         https://firebase.google.com/docs/web/setup#available-libraries -->
-    <script src="https://www.gstatic.com/firebasejs/7.17.2/firebase-analytics.js"></script>
-    
-    <script>
-      // Concerned Citizen,
-      // We know what you're asking yourself: "Did these brain dead Covid Watch devs just check their API keys in to
-      // an open source repository?". The answer is of course, yes we did. But don't worry, these are not "API keys"
-      // in the normal sense of the word. They should rather be thought of as configuration URLs that tell our app
-      // which Firebase infrastructure it should talk to. Firebase handles security in other ways, see the following
-      // Stack Overflow link for more info: https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public
-
-      var firebaseConfig = {
-        apiKey: "AIzaSyCnaCYo8H_EfJNAbQkEQJeBypqnAHvtCNs",
-        authDomain: "permission-portal.firebaseapp.com",
-        databaseURL: "https://permission-portal.firebaseio.com",
-        projectId: "permission-portal",
-        storageBucket: "permission-portal.appspot.com",
-        messagingSenderId: "1090793610922",
-        appId: "1:1090793610922:web:24a8e68a7107347b1bfad3",
-        measurementId: "G-BDXQC3ND00"
-      };
-      // Initialize Firebase
-      firebase.initializeApp(firebaseConfig);
-      firebase.analytics();
-    </script>
+      <script src="https://www.gstatic.com/firebasejs/7.17.2/firebase-app.js"></script>
```

Comment:
We do the equivalent of this with the `import app from 'firebase/app'` in `client/src/store/firebase.js`